server/src/websocket.js

```
@@ -0,0 +1,86 @@
+    ws.on('message', function (data) {
+        var msg = JSON.parse(data['utf8Data']);
+
+        if (!("type" in msg)) {
+            msg.data = "Missing type";
+            ws.send(JSON.stringify(msg));
+            return;
+        }
+
+        switch (msg["type"]) {
+            case "workspaces":
+                const workspacePath = path.join(process.cwd(), 'workspace');
```

Biome (2.4.7)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)